README.md

```diff
@@ -45,7 +45,7 @@ information. When a request comes in on the request phase url the relevant
 strategy will receive the `handle_request!` call.
 
 In some cases (default) the application using Ueberauth is responsible for
-implementing the request phase. That is, you should setup a route to receive
+implementing the request phase. That is, you should set up a route to receive
```

Comment:
`setup` is an actual word, so we should live this as it it

Reply:
@tajchumber I believe this actually the proper usage.  You "set up" a computer. Your computer & keyboard are your work "setup". The former a verb (like the usage here) and the latter a noun.

You "set up" a "setup".

Reply:
Yeah! `Set up` is a verb while `setup` is a substantive.